### PR TITLE
Issue #27: Improve README

### PR DESCRIPTION
**Pull request by dmitriim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,55 @@ An admin tool that allows you to control a block layout across units.
...
+- Weight - what block weight will be on those pages? 
+- Config data - does block have any config? Could be applied here (but should be exact string of config that is stored in DB if you would add the block manually)? 
+- Visible? - is block visible when added? 
+- Add another instance (if exists)? - if the ame block already exists, should we add another instance of that block? 
+- Update instance (if exists)? - if the block already exists, should we update the block by new settings? 
+- Reposition instance (if exists)? - if the block already exists, should we reposition the block?
+- Show in subcontexts - should the block be displayed on course subpages (e.g. course page and activity pages)?
+- Page type pattern - what page should the block be displayed? E.g. course-view-*, mod-assign-view and etc.
+                                                                                                           
+Multiple blocks can be configured to be applied. Once all done and form is submitted. The set of blocks will be applied
```
Comment (TomoTsuyuki) [inline README.md:75]: I was confused about `Add new line` and `Apply set of blocks` button.
Also better to add what `Set of blocks` textarea is.

**Review by TomoTsuyuki — COMMENTED** [review]
As I am a new user for this plugin, I wondered some settings.
I added some comments where I was confused / couldn't understand.

File: README.md [hunk 2]
```diff
@@ -26,6 +26,55 @@ An admin tool that allows you to control a block layout across units.
...
+
+### Set up blocks ### 
+This features helps to create blocks in bulk through selected course categories (for example add a calendar block to every course 
+for a selected category).
+
+To do  that, navigate to Site administration > Plugins > Admin tools > Blocks Manager > Set up blocks
+
+The form let you to select: 
+
+- Region - what region to add a block to?
```
[the remark this answers is not in the document]
Comment (dmitriim) [reply README.md:63]: Regions are positions where blocks can be added on a page. Theme defines what regions are exist. From 4.0 there is only one region in core boost theme, but custom theme can have more regions.

[the remark this answers is not in the document]
Comment (dmitriim) [reply README.md:63]: image

File: README.md [hunk 3]
```diff
@@ -26,6 +26,55 @@ An admin tool that allows you to control a block layout across units.
...
+- Block - what block to add? 
+- Categories - what categories to apply the block? 
+- Weight - what block weight will be on those pages? 
+- Config data - does block have any config? Could be applied here (but should be exact string of config that is stored in DB if you would add the block manually)?
```
[the remark this answers is not in the document]
Comment (dmitriim) [reply README.md:67]: This is tricky. I have a whole new issue for managing that. Currently it's silly. What we add -> what will be applied. I will sort it as part of https://github.com/catalyst/moodle_tool_blocksmanager/issues/35

File: README.md [hunk 4]
```diff
@@ -26,6 +26,55 @@ An admin tool that allows you to control a block layout across units.
...
+
+The form let you to select: 
+
+- Region - what region to add a block to?
+- Block - what block to add? 
+- Categories - what categories to apply the block? 
+- Weight - what block weight will be on those pages? 
+- Config data - does block have any config? Could be applied here (but should be exact string of config that is stored in DB if you would add the block manually)? 
+- Visible? - is block visible when added? 
+- Add another instance (if exists)? - if the ame block already exists, should we add another instance of that block?
```
[the remark this answers is not in the document]
Comment (dmitriim) [reply README.md:69]: Fixed

File: README.md (hunk 1 above)
Comment (dmitriim) [reply README.md]: Yeah. I know UI is confusing. I tried to improve the description, please check.

File: README.md [hunk 5]
```diff
@@ -26,6 +26,55 @@ An admin tool that allows you to control a block layout across units.
...
+- hiding 
+- moving
+
+To create set of rules to lock blocks, navigate to Site administration > Plugins > Admin tools > Blocks Manager > Blocks management    
+
+### Region management ###
+This feature gives an ability to lock a whole region through a selected course category. That means all blocks in the 
+matching region will be locked from performing configured actions (expect for users with 'tool/blocksmanager:bypasslocking'). 
+
+Actions that could be locked:
```
[the remark this answers is not in the document]
Comment (dmitriim) [reply README.md]: Good catch. Fixed

**Pull request merged**